app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class ReservationsController < ApplicationController
+
+  def create
+    @property = Property.find_by(id: params[:property_id])
+
+    start_date = Date.strptime(params[:reservation]['start_date'], "%m/%d/%Y")
+    end_date = Date.strptime(params[:reservation]['end_date'], "%m/%d/%Y")
+
+    @reservation = current_user.reservations.new(start_date: start_date, end_date: end_date)
```

Comment:
Creation of `reservation` should be done after it is proven that current user is not the property owner.